lib/Layout.js

```diff
@@ -132,6 +132,34 @@
 
 'use strict';
 
+const Buffer = require('buffer').Buffer;
+
+/* Check if a value is a Uint8Array.
+ *
+ * @ignore */
+function checkUint8Array(b) {
+  if (!(b instanceof Uint8Array)) {
+    throw new TypeError('b must be a Uint8Array');
+  }
+}
+exports.checkUint8Array = checkUint8Array;
+
+/* Create a Buffer instance from a Uint8Array.
+ *
+ * @ignore */
+function uint8ArrayToBuffer(b) {
+  checkUint8Array(b);
+  let buffer = new Buffer(b.buffer, b.byteOffset, b.length);
+
+  // Node v4 doesn't respect byteOffset and length arguments
+  if (buffer.length > b.length) {
+    buffer = buffer.slice(b.byteOffset, b.byteOffset + b.length);
+  }
```

Comment:
Without bumping the min node version, we need a little polyfill to account for the fact that node v4 doesn't have support for the `byteOffset` and `length` arguments to the `Buffer` constructor.

Reply:
As noted in the review summary, I'm going to remove v4 support so you won't need this.